src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +23,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
Remember to add a private default constructor to the `HibernateUtil` class in order to prevent creating `HibernateUtil` objects. This is a common singleton pattern practice.